res/res_usbradio.c

```diff
@@ -421,15 +459,15 @@ struct usb_device *ast_radio_hid_device_init(const char *desired_device)
 	usb_find_devices();
 	for (usb_bus = usb_busses; usb_bus; usb_bus = usb_bus->next) {
 		for (dev = usb_bus->devices; dev; dev = dev->next) {
-			if ((dev->descriptor.idVendor
-				 == C108_VENDOR_ID) &&
+			if (((dev->descriptor.idVendor == C108_VENDOR_ID) &&
 				(((dev->descriptor.idProduct & 0xfffc) == C108_PRODUCT_ID) ||
 				 (dev->descriptor.idProduct == C108B_PRODUCT_ID) ||
 				 (dev->descriptor.idProduct == C108AH_PRODUCT_ID) ||
 				 (dev->descriptor.idProduct == C119A_PRODUCT_ID) ||
 				 (dev->descriptor.idProduct == C119B_PRODUCT_ID) ||
 				 ((dev->descriptor.idProduct & 0xff00) == N1KDO_PRODUCT_ID) ||
-				 (dev->descriptor.idProduct == C119_PRODUCT_ID))) {
+				 (dev->descriptor.idProduct == C119_PRODUCT_ID))) ||
+				 is_user_device(dev)) {

```

Comment:
Same comment as above re: spacing and the need for a new function to match our known/supported/blessed vid/pid's.